### PR TITLE
core/config: rename block_xpub column to block_pub

### PR DESCRIPTION
**Pull request by tessr**
This fulfills one of my old todos and prepares for grpc (`chainjson.HexBytes` is not compatible with protobufs).

### CODE REVIEW

File: core/config/config.go [hunk 1]
```diff
@@ -44,23 +43,23 @@ type Config struct {
...
 	Quorum               int
 	MaxIssuanceWindow    time.Duration
 }
 
 type BlockSigner struct {
-	AccessToken string             `json:"access_token"`
-	Pubkey      chainjson.HexBytes `json:"pubkey"`
-	URL         string             `json:"url"`
+	AccessToken string `json:"access_token"`
+	Pubkey      []byte `json:"pubkey"`
```
Comment (erykwalder) [inline core/config/config.go]: Until we have grpc, we should probably leave these as hex bytes. I think otherwise requests will have to use base64.

Comment (tessr) [inline core/config/config.go]: Ah, I wonder if that's what's making the dashboard tests complain.

Comment (tessr) [inline core/config/config.go]: (Hmm, turns out, no, they were genuine lint errors. Regardless, I'll turn the pubkeys back into hex bytes.)

**Comment by tessr** [comment]
This no longer changes the type. PTAL

**Comment by erykwalder** [comment]
LGTM

**Pull request merged**